samples/dashboards/demo/election/demo.css

```diff
@@ -0,0 +1,571 @@
+}
+
+#election-map .highcharts-tooltip-box {
+    fill: #9eb8f9;
+    fill-opacity: 1;
+    stroke: none;
+}
+
+/* Custom map tooltip */
+
+#election-map .map-tooltip caption {
+    font-size: 13px;
+    font-weight: bold;
+    padding-bottom: 5px;
+}
+
+#election-map .map-tooltip td,
+#election-map .map-tooltip th {
+    background-color: var(--color-tooltip-background);
+    text-align: left;
+    padding: 3px;
+}
+
+/* Electoral votes */
+#election-map .map-tooltip td:nth-of-type(2) {
+    text-align: center;
+}
+
+/* Winning candidate */
+#election-map tr:nth-of-type(4) th {
+    font-size: 12px;
+    text-align: center;
+}
+
+/* *
+ *
+ *  Election chart (history)
+ *
+ * */
+
+#election-chart .highcharts-title {
+    fill: var(--color-title);
+    letter-spacing: 1.5px;
+    font-size: 1.2em;
+    text-transform: capitalize;
+}
+
+#election-chart .highcharts-axis {
+    fill: var(--color-axis);
+}
+
+#election-chart .highcharts-axis-line,
+#election-chart .highcharts-tick {
+    stroke: var(--color-axis);
+}
+
+#election-chart .highcharts-axis-labels,
+#election-chart .highcharts-axis-title {
+    fill: var(--color-axis-labels);
+}
+
+#election-chart .highcharts-tooltip {
+    fill: none;
+    filter: none;
+    stroke: none;
+}
+
+/* Readable inside and outside column */
+#election-chart .highcharts-data-label text {
+    fill: #fff;
+    stroke: #000;
+    paint-order: stroke;
+    stroke-width: 2px;
+}
+
+#election-chart .highcharts-tooltip text {
+    fill: var(--color-tooltip-text);
+}
+
+#election-chart .highcharts-tooltip-box {
+    fill: var(--color-tooltip-background);
+    fill-opacity: 1;
+    stroke: none;
+}
+
+.highcharts-legend-item > text {
+    fill: var(--color-text);
+}
+
+/* *
+ *
+ *  Selection Grid
+ *
+ * */
+
+#election-grid .highcharts-dashboards-component {
+    background-color: var(--color-background);
+}
+
+#election-grid .highcharts-datagrid-column-header {
+    font-size: 12px;
+}
+
+#election-grid .highcharts-datagrid-row.hovered {
+    background-color: var(--color-hover-background);
+    border-color: var(--color-hover-border);
+    color: var(--color-hover-text);
+}
+
+/* Styling of first row (national results) */
+#election-grid .highcharts-datagrid-row:nth-of-type(1) {
+    font-weight: bold;
+}
+
+/* Capitalize state names (first column) */
+#election-grid .highcharts-datagrid-cell:nth-child(1) {
+    text-transform: capitalize;
+}
+
+/* Candidate votes columns */
+#election-grid .highcharts-datagrid-cell:nth-child(4),
+#election-grid .highcharts-datagrid-cell:nth-child(5) {
+    min-width: 70px;
+}
+
+/* Total votes column */
+#election-grid .highcharts-datagrid-cell:nth-child(6) {
+    min-width: 100px;
+}
+
+/* Safari fix for manual dark mode */
+/* stylelint-disable-next-line max-line-length */
+.highcharts-dark #election-map .highcharts-series-1 .highcharts-data-label text {
+    paint-order: stroke;
+    stroke-width: 2px;
+}
+
+/* Dashboards layout */
+
+/* LARGE */
+@media (max-width: 1200px) {
```

Comment:
I wonder if it wouldn't be better to switch `@media` queries to `@container` in future demos. In this case, responsiveness would result from the width of the board, not the entire page. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries

See others' input from the accounting demo: https://github.com/highcharts/highcharts/pull/20220#issuecomment-1958909467

And then mobile view in utils will also work.